pkg/collector/python/check.go

```diff
@@ -212,6 +214,9 @@ func (c *PythonCheck) Configure(data integration.Data, initConfig integration.Da
 
 		res := C.get_check_deprecated(rtloader, c.class, cInitConfig, cInstance, cAgentConfig, cCheckID, cCheckName, &check)
 		if res == 0 {
+			if rtLoaderError != nil {
+				return fmt.Errorf("could not invoke '%s' python check constructor. First got:\n%sthen:\n%s", c.ModuleName, rtLoaderError, getRtLoaderError())
```

Comment:
The log message should mention what are the first and second error.

something like this maybe ?

```
could not invoke '%s' python check constructor. New init API returned:\n%sDeprecated Init API returned:\n%s
```

WDYT ?

Reply:
Sounds good. I used `constructor` instead of `init`, let me know what you think.

Reply:
I think you forgot to push you changes. `constructor` sound better indeed.

Reply:
Yep sorry, pushed now.